hack/tools/go.mod

```diff
@@ -3,26 +3,10 @@ module sigs.k8s.io/cluster-api/hack/tools
 go 1.15
 
 require (
-	github.com/drone/envsubst v1.0.3-0.20200804185402-58bc65f69603 // indirect
-	github.com/golang/mock v1.4.3
-	github.com/golangci/golangci-lint v1.17.1
-	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
+	github.com/golangci/golangci-lint v1.37.1
+	github.com/jteeuwen/go-bindata v3.0.7+incompatible
 	k8s.io/code-generator v0.18.2
 	sigs.k8s.io/controller-tools v0.3.0
-	sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e // indirect
+	sigs.k8s.io/kustomize/kustomize/v3 v3.10.0
 	sigs.k8s.io/testing_frameworks v0.1.1
 )
-
-replace (
```

Comment:
removing it: [golangci/golangci-lint#595](https://github.com/golangci/golangci-lint/issues/595) is fixed.